static/report/js/predictions.js

```diff
@@ -0,0 +1,40 @@
+
+var predictedOffset = 0;
+
+function predictForward() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['predictForward' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/m-kozak/cgm-remote-monitor/pullRequest?prid=1196151)